backend/fms_core/serializers.py

```diff
@@ -590,32 +589,48 @@ class Meta:
         fields = "__all__"
 
 class DatasetSerializer(serializers.ModelSerializer):
-    files = serializers.SerializerMethodField()
+    # files = serializers.SerializerMethodField()
     released_status_count = serializers.SerializerMethodField()
     blocked_status_count = serializers.SerializerMethodField()
-    latest_release_update = serializers.SerializerMethodField()
     readset_count = serializers.SerializerMethodField()
     archived_comments = ArchivedCommentSerializer("archived_comments", many=True)
+    latest_release_update = serializers.SerializerMethodField()
+    released_by = serializers.SerializerMethodField()
+    validation_status = serializers.SerializerMethodField()
+    validation_status_update_timestamp = serializers.SerializerMethodField()
+    validated_by = serializers.SerializerMethodField()
 
     class Meta:
         model = Dataset
-        fields = ("id", "external_project_id", "run_name", "lane", "files", "released_status_count", "blocked_status_count", "latest_release_update", "validation_status", "project_name", "metric_report_url", "readset_count", "archived_comments")
+        fields = "__all__"
 
     def get_files(self, obj):
         return DatasetFile.objects.filter(readset__dataset=obj.id).values_list("id", flat=True)
 
     def get_released_status_count(self, obj):
         return Readset.objects.filter(dataset=obj.id, release_status=ReleaseStatus.RELEASED).count()
-    
+
     def get_blocked_status_count(self, obj):
         return Readset.objects.filter(dataset=obj.id, release_status=ReleaseStatus.BLOCKED).count()
-    
+
     def get_latest_release_update(self, obj):
         return Readset.objects.filter(dataset=obj.id).aggregate(Max("release_status_timestamp"))["release_status_timestamp__max"]
-    
+
     def get_readset_count(self, obj):
         return Readset.objects.filter(dataset=obj.id).count()
 
+    def get_validation_status_update_timestamp(self, obj):
```

Comment:
Keep the same naming pattern as release. Both are so similar i would prefer them to have a naming that reflect that.